canonical-web-and-design/jaas.ai
# Perf improvements

## Done

- Add preconnect to Marketo
- Omit sourcemap url from production build
- Defer app.js in main layout

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Sanity check the code
